.prettierrc.json

```diff
@@ -0,0 +1,8 @@
+{
+  "semi": false,
+  "trailingComma": "all",
+  "singleQuote": true,
+  "printWidth": 100,
+  "tabWidth": 2,
+  "plugins": ["prettier-plugin-organize-imports"]
```

Comment:
I use this prettier plugin with all of my projects - it keeps files nice and tidy